### PR TITLE
[Octopus] launch update

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by adelasofia** [comment]
Hold for Mike Guerette's approval

**Comment by adelasofia** [comment]
@LightGuard Approved by Mike, please review :smiley:

File: products/rhel/download.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 To download Red Hat Enterprise Linux Developer Suite, which includes Red Hat Enterprise Linux 7 server, a collection of development tools, and link:#{site.base_url}/products/rhel/overview/[much more], you must have an account and need to accept the terms and conditions of the Red Hat Developer Program which provides $0 subscriptions for development use only. link:#{site.base_url}/faq[Read more] about the Red Hat Developers Program.
 
-==== link:https://developers.stage.redhat.com/download-manager/file/rhel-server-7.2-x86_64-dvd.iso[Download Red Hat Enterprise Linux 7]
+==== link:#{site.base_url}/download-manager/file/rhel-server-7.2-x86_64-dvd.iso[Download Red Hat Enterprise Linux 7]
```
Comment (paulrobinson) [inline products/rhel/download.adoc]: You need to use 'site.download_manager_base_url' not 'site.base_url'. Currently the above renders as: http://developers-pr.stage.redhat.com/pr/941/build/609/download-manager/file/rhel-server-7.2-x86_64-dvd.iso

Comment (paulrobinson) [inline products/rhel/download.adoc]: This happens elsewhere in this PR too.

Comment (paulrobinson) [inline products/rhel/download.adoc]: FYI: The Blinkr report spotted this.

**Comment by LightGuard** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
@paulrobinson @LightGuard I've fixed the download links, I believe this is ready to be merged

**Comment by adelasofia** [comment]
The new content updates have been approved by Mike G.

**Comment by LightGuard** [comment]
@adelasofia There's still an issue in blinkr which is new.

Page: http://developers-pr.stage.redhat.com/pr/941/build/615/articles/rhel-what-you-need-to-know/
URL: http://developers-pr.stage.redhat.com/pr/941/build/615/download-manager/link/1350474

File: _layouts/product-base.html.slim [hunk 2]
```diff
@@ -15,7 +15,7 @@ section(class="product-header #{page.product.id} #{page.product.product_category
       h3#product #{page.product.description}
 
 section(class="product-graphics #{page.product.product_category}")
-  - if page.product.id != 'rhel'
+  - if !page.url.include?('/products/rhel/get-started')
```
Comment (LightGuard) [inline _layouts/product-base.html.slim]: Using `unless page.url.include? ...` may read a bit better than `if !page...`

Comment (adelasofia) [inline _layouts/product-base.html.slim]: You're right, I've just sent the fix.

**Pull request merged**